content/en/tracing/guide/add_span_md_and_graph_it.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Add span tags and slice and dice them using Trace Analytics
+kind: guide
+disable_toc: true
+further_reading:
+- link: "tracing/setup/"
+  tag: "Documentation"
+  text: "Learn how to setup APM tracing with your application"
+- link: "tracing/visualization/services_list/"
+  tag: "Documentation"
+  text: "Discover the list of services reporting to Datadog"
+- link: "tracing/visualization/resource"
+  tag: "Documentation"
+  text: "Dive into your resource performance and traces"
+- link: "tracing/visualization/trace"
+  tag: "Documentation"
+  text: "Understand how to read a Datadog Trace"
+---
+
+{{< img src="tracing/guide/add_span_md_and_graph_it/span_md_6.gif" alt="Analytics View" responsive="true" style="width:90%;">}}
+
+Datadog APM allows you to customize your traces to include any additional information you might need to maintain observability into your business. In this example, a customer ID is added to traces allowing the customers that have the slowest performance to be identified. Customization of traces is based on tags that seamlessly integrate APM with the rest of Datadog and come in the form of `key:value` pairs of metadata added to spans.
+
+## Instrument your code with custom metadata
+
+1) **Follow the example to get your code instrumented**.
+
+Depending on the programming language you are you using, you’ll need to set the metadata to add to your spans differently.
+
+**Note**: take note of the service and [resource names][1] you are working on, these will come in handy later. In this example, the service is the Ruby server `web-store` and the resource (endpoint) is `ShoppingCartController#checkout`.
+
+{{< tabs >}}
+{{% tab "Java" %}}
+
+The Datadog UI uses tags to set span level metadata. A full list of the metadata can be found in the [Datadog][1] and [OpenTracing][2] APIs.
+
+Custom metadata may be set for auto-instrumentation by grabbing the active span out of the global tracer and setting a tag with `setTag`.
+
+```java
+import io.opentracing.Tracer;
+import io.opentracing.util.GlobalTracer;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet
+class ServletImpl extends AbstractHttpServlet {
```

Comment:
I'm changing all the examples to use `ShoppingCartController`. This makes everything consistent.